tests/OSPSuite.Infrastructure.Tests/Import/MixColumnsDataFormatSpecs.cs

```diff
@@ -0,0 +1,119 @@
+﻿using FakeItEasy;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Import;
+using OSPSuite.Helpers;
+using OSPSuite.Infrastructure.Import.Core;
+using OSPSuite.Infrastructure.Import.Core.DataFormat;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OSPSuite.Infrastructure.Import
+{
+   public abstract class concern_for_MixColumnsDataFormat : ContextSpecification<MixColumnsDataFormat>
+   {
+      protected IUnformattedData _rawData;
+      protected ColumnInfoCache _columnInfos;
+      protected IReadOnlyList<MetaDataCategory> _metaDataCategories;
+      protected IEnumerable<string> _headers { get; set; } = new[] { "Time [h]", "Concentration [mol]", "Error [mol]" };
+
+      protected override void Context()
+      {
+         sut = new MixColumnsDataFormat();
+         _rawData = A.Fake<IUnformattedData>();
+         A.CallTo(() => _rawData.GetHeaders()).ReturnsLazily(() => _headers);
+         A.CallTo(() => _rawData.GetColumnDescription(A<string>.Ignored)).Returns(new ColumnDescription(0) { Level = ColumnDescription.MeasurementLevel.Numeric });
+         _columnInfos = new ColumnInfoCache
+         {
+            new ColumnInfo() { DisplayName = "Time", Name ="Time" },
+            new ColumnInfo() { DisplayName = "Concentration", Name = "Concentration"},
+            new ColumnInfo() { DisplayName = "Error", Name = "Error", IsMandatory = false, RelatedColumnOf = "Concentration"}
+         };
+         _columnInfos["Time"].SupportedDimensions.Add(DimensionFactoryForSpecs.TimeDimension);
+         _columnInfos["Concentration"].SupportedDimensions.Add(DimensionFactoryForSpecs.ConcentrationDimension);
+         _columnInfos["Error"].SupportedDimensions.Add(DimensionFactoryForSpecs.ConcentrationDimension);
+         _metaDataCategories = new List<MetaDataCategory>()
+         {
+            new MetaDataCategory() { Name = "Organ" },
```

Comment:
You do not need the () when initializing an object with props. It's just noise. Use reshaper to remove them